asminc/telestrat.inc

```diff
@@ -173,6 +176,7 @@ XZAP             = $46
 XSHOOT           = $47
 XMKDIR           = $4B          ; create a folder. Only available in telemon 3.x
 XRM              = $4D          ; remove a folder or a file. Only available in telemon 3.x
+XMALLOC          = $5B          ; only in telemon 3.x
```

Comment:
Maybe you want to / should generally settle on 'telemon' vs. 'TELEMON' and make all occurances consistent.